bitnami/rabbitmq/templates/secrets.yaml

```diff
@@ -1,29 +1,18 @@
 {{- if or (not .Values.auth.existingErlangSecret) (not .Values.auth.existingPasswordSecret) }}
+{{- $secretName := include "common.secrets.name" (dict "existingSecret" .Values.auth.existingSecret "context" $)}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "rabbitmq.fullname" . }}
+  name: {{ $secretName }}
```

Comment:
This is tricky here, as the change is not exactly the same as the PR I shared with you. Bear in mind, that this secret will be created **if either** `existingErlangSecret` or `existingPasswordSecret` are empty.

This means, that if you **only provide one** existing secret, say `existingErlangSecret`, the installation will fail as Helm will try to create a new Secret that will be named exactly like the one that already exists, which will lead to failure. 

```bash
$ kubectl create secret generic abc --from-literal rabbitmq-password=ABCDEFG12345 --from-literal rabbitmq-erlang-cookie=ABCDEFG12345
secret/abc created

$ helm install delta bitnami/rabbitmq --set auth.existingSecret=abc
Error: rendered manifests contain a resource that already exists. Unable to continue with install: Secret "abc" in namespace "default" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "delta"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "default"
```

This case is more similar to Odoo:

https://github.com/bitnami/charts/blob/master/bitnami/odoo/templates/secrets.yaml
```suggestion
  name: {{ include "rabbitmq.fullname" . }}
```